react-hello-world-master/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "axios": "^0.26.1",
     "debug": "^4.3.4",
+    "eslint-plugin-react": "^7.29.4",
```

Comment:
add in readme.md information what kind of extension you need to add to use it and exemplary screenshot